admin_honeypot/models.py

```diff
@@ -2,10 +2,15 @@
 from django.utils.translation import ugettext_lazy as _
 from admin_honeypot import listeners
 
+try:
+    from django.db.models import GenericIPAddressField
+except:
+    from django.db.models import IPAddressField as GenericIPAddressField
```

Comment:
How about adding a comment like:

    # Removed in Django 1.9

So in the future, you know when you can delete this fallback.

Also, you should **never** catch all exceptions. Just do the `ImportError` or whatever the actual exception is.